frontend/webpack.config.ts

```diff
@@ -39,15 +39,24 @@ const extractCSS = new MiniCssExtractPlugin({
   // See https://github.com/webpack-contrib/mini-css-extract-plugin/issues/250
   ignoreOrder: true,
 });
+const virtualModules = new VirtualModulesPlugin();
+
 const getVendorModuleRegExp = (vendorModules: string[]) =>
   new RegExp(`node_modules\\/(${vendorModules.map((m) => _.escapeRegExp(m)).join('|')})\\/`);
-const virtualModules = new VirtualModulesPlugin();
+
 const overpassTest = /overpass-.*\.(woff2?|ttf|eot|otf)(\?.*$|$)/;
-const sharedPluginTest = getVendorModuleRegExp(Object.keys(sharedPluginModules));
-const sharedPatternFlyCoreTest = getVendorModuleRegExp([
-  '@patternfly/react-core',
-  '@patternfly/react-table',
-]);
+
+const sharedPluginTest = getVendorModuleRegExp(
+  sharedPluginModules.map((moduleName) =>
+    // Console provides specific PatternFly 4 shared modules to its dynamic plugins.
+    // These are represented in Console webpack compilation as @patternfly-4/* modules.
+    moduleName.startsWith('@patternfly/')
+      ? moduleName.replace(/^@patternfly\//, '@patternfly-4/')
+      : moduleName,
+  ),
+);
+
+const sharedPatternFlyCoreTest = /node_modules\/@patternfly(-\S)?\//;
```

Comment:
TODO need to split this into PF5 modules (treeshaked) used by Console vs. PF4 modules (non-treeshaked) provided as shared modules to dynamic plugins.